rollup.config.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-unresolved */
+const dynamicImportVars = require('@rollup/plugin-dynamic-import-vars');
+const terser = require('@rollup/plugin-terser');
+
+module.exports = {
+  preserveEntrySignatures: true,
+  input: [
+    './scripts/scripts.js',
+    // is already imported by script.js.
+    // There is no need to generate a bundle for it (ignoring 404.html)
+    // './scripts/lib-franklin.js',
```

Comment:
The issue with creating two bundles for both scripts.js and lib-franking.js was that they have some common dependencies, and rollup tried to extract these to guarantee that both bundles can run standalone without any duplicate code being loaded. but currently scripts and franklin were loaded on every page, (webpack has an option for this: https://webpack.js.org/concepts/entry-points/#entrydescription-object but i didn't find something similar for rollup) and i wanted to have less number of requests as possible to test its effect. so i removed the script tag for franklin. the code is being imported by scripts.js anyways.